distribution/docs/src/main/resources/_contents/_catalog-frameworks/default-attribute-values-contents.adoc

```diff
@@ -0,0 +1,69 @@
+
+==== Default Attribute Values
```

Comment:
This looks very similar (though worded differently) to the `attribute-type-definition-contents.adoc` file. Again, any way to extract?